irohad/main/impl/block_loader_init.hpp

```diff
@@ -34,24 +34,25 @@ namespace iroha {
        * @param storage - used to retrieve blocks
```

Comment:
Please, update all comments in this file. For example, here new parameter is used